options/options.js

```diff
@@ -8,6 +8,13 @@ setupRadioButtons();
 enforceInputRange($('#popupWidth'));
 setTimeout(splitLongTooltips);
 
+if (CHROME >= 3167 && CHROME <= 3704) {
```

Comment:
Since this condition is used multiple times, let's pull it out as a constant. In `messaging,js`:
```js
const CHROME_HAS_BORDER_BUG = CHROME >= 3167 && CHROME <= 3704;
```
To use the condition:
```js
if (CHROME_HAS_BORDER_BUG) {
  ...
}
```